From: martin f. krafft <madduck@debian.org>
Subject: Fix compiler/linker flag in Makefile

Cherry-pick 8efb9d1 from upstream to replace -z now with -Wl,-z,now

Debian bug: #595290

Signed-off-by: martin f. krafft <madduck@debian.org>

---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index e2c65a5..0cc9a87 100644
--- a/Makefile
+++ b/Makefile
@@ -167,7 +167,7 @@ mdmon.O2 : $(MON_SRCS) mdadm.h mdmon.h
 
 # use '-z now' to guarantee no dynamic linker interactions with the monitor thread
 mdmon : $(MON_OBJS)
-	$(CC) $(LDFLAGS) $(MON_LDFLAGS) -z now -o mdmon $(MON_OBJS) $(LDLIBS)
+	$(CC) $(LDFLAGS) $(MON_LDFLAGS) -Wl,-z,now -o mdmon $(MON_OBJS) $(LDLIBS)
 msg.o: msg.c msg.h
 
 test_stripe : restripe.c mdadm.h
-- 
tg: (ef9f23f..) fixes/linker-flag-z-now (depends on: master)
